src/test/scala/spray/json/JsonParserSpec.scala

```diff
@@ -144,6 +144,13 @@ class JsonParserSpec extends Specification {
           |{}x
           |  ^
           |""".stripMargin
+
+      "reject numbers which are too big / have too high precision" in {
+        val settings = JsonParserSettings.default.withMaxNumberCharacters(5)
+        errorMessage("123.4567890", settings) ===
+          "Number too long:The number starting with '123.4567890' had 11 characters which is more than the allowed limit " +
```

Comment:
No space between short and long message? Outside of the scope of this PR though ;).

Reply:
Yep, found that weird as well ;)